pkgs/applications/audio/fmit/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, mkDerivation, fetchFromGitHub, fftw, qtbase, qtmultimedia, qmake, itstool, wrapQtAppsHook
-, alsaSupport ? true, alsa-lib ? null
-, jackSupport ? false, libjack2 ? null
-, portaudioSupport ? false, portaudio ? null }:
+, alsaSupport ? true, alsa-lib
+, jackSupport ? false, libjack2
+, portaudioSupport ? false, portaudio }:
 
 assert alsaSupport -> alsa-lib != null;
 assert jackSupport -> libjack2 != null;
```

Comment:
What are we doing with those? Are you gonna remove them, too or will this be another cleanup PR?

Reply:
Using `null` to indicate absence of a dependency can still make sense, which can be done with `.override` or overlays. So these checks would still give early errors in some cases.

Personally I'm not sure how useful these assertions are though, maybe we should think about removing them at some point.

Reply:
I am personally for removing them.